dataprofiler/profilers/profile_builder.py

```diff
@@ -955,10 +986,6 @@ def _restore_data_labelers(self, data_labeler=None):
                 self.options.set({"data_labeler.is_enabled": False})
                 self.options.set({"data_labeler.data_labeler_object": data_labeler})
 
-            except Exception as e:
```

Comment:
Why was this removed?

Reply:
nvm

Reply:
NVM. I think this is actually broken code.  I believe there's a path which makes it not a duplicate.

Could definitely be refactored.

Essentially, if DP is not None and an error was raised, it would try to set it again in the first case and re raise I beleive.

Reply:
Maybe it is ideal to re-raise bc we shouldn't be setting something that fails.